modules/couchbase/src/main/java/org/testcontainers/couchbase/CouchbaseContainer.java

```diff
@@ -67,12 +61,11 @@
 @AllArgsConstructor
 public class CouchbaseContainer extends GenericContainer<CouchbaseContainer> {
 
-    public static final String VERSION = "5.1.0";
+    public static final String VERSION = "5.1.1";
```

Comment:
any reason we stick to 5.1.1 here? 5.5.0 is out. Is there a reason it is not using latest if not otherwise specified explicitly by the user?

Reply:
api compatibility. But this release will have breaking changes anyway for couchbase containers, so why not

Reply:
👍